app/Commands/DeployCommand.php

```diff
@@ -159,11 +161,23 @@ protected function maybeCreateDatabase(Server $server, Site $site)
         $this->forge->updateSiteEnvironmentFile($server->id, $site->id, $env);
     }
 
+    protected function maybeOutput(string $key, string $value): void
+    {
+        $prefix = $this->option('ci-prefix');
```

Comment:
Instead of making the prefix changeable, we should just stick to something like `forge_previewer` or similar.

Reply:
Thanks @ryangjchandler, I've updated this and removed the option.